tabs/receiver.js

```diff
@@ -120,21 +142,24 @@ TABS.receiver.initialize = function (callback) {
         $('.deadband input[name="deadband"]').val(RC_deadband.deadband);
 
         // generate bars
+        // woga65: including AUX- and/or variable pitch specific channel names
         var bar_names = [
                 chrome.i18n.getMessage('controlAxisRoll'),
                 chrome.i18n.getMessage('controlAxisPitch'),
                 chrome.i18n.getMessage('controlAxisYaw'),
-                chrome.i18n.getMessage('controlAxisThrottle')
+                chrome.i18n.getMessage('controlAxisThrottle'),
+                'Aux 1 [1]',
```

Comment:
Please just use channel numbers, not **aux** numbers. The 4 channels after AETR should be: 
CH 5
CH 6
CH 7 or Collective [7]
CH 8 or Gyro gain [8]

It needs to be consistent with the current release.
![image](https://github.com/iNavFlight/inav-configurator/assets/17590174/b639acfe-e16d-4d70-a0d5-44c1a17f8d13)